scripts/get-shapefiles.py

```diff
@@ -28,6 +29,9 @@
     import urllib2
     import urlparse
 
+__version__ = '2.0.0'
```

Comment:
It was added just to mention that there have been modifications in the code. No problem, if you consider to remove this, I’ll do.